prometheus-metrics-instrumentation-dropwizard/src/test/java/io/prometheus/metrics/instrumentation/dropwizard/DropwizardExportsTest.java

```diff
@@ -0,0 +1,319 @@
+        };
+    Gauge<Boolean> booleanGauge =
+        new Gauge<Boolean>() {
+          @Override
+          public Boolean getValue() {
+            return true;
+          }
+        };
+
+    metricRegistry.register("double.gauge", doubleGauge);
+    metricRegistry.register("long.gauge", longGauge);
+    metricRegistry.register("integer.gauge", integerGauge);
+    metricRegistry.register("float.gauge", floatGauge);
+    metricRegistry.register("boolean.gauge", booleanGauge);
+
+    String expected =
+        "# TYPE boolean_gauge gauge\n"
+            + "# HELP boolean_gauge Generated from Dropwizard metric import (metric=boolean.gauge, type=io.prometheus.metrics.instrumentation.dropwizard.DropwizardExportsTest$5)\n"
+            + "boolean_gauge 1.0\n"
+            + "# TYPE double_gauge gauge\n"
+            + "# HELP double_gauge Generated from Dropwizard metric import (metric=double.gauge, type=io.prometheus.metrics.instrumentation.dropwizard.DropwizardExportsTest$2)\n"
+            + "double_gauge 1.234\n"
+            + "# TYPE float_gauge gauge\n"
+            + "# HELP float_gauge Generated from Dropwizard metric import (metric=float.gauge, type=io.prometheus.metrics.instrumentation.dropwizard.DropwizardExportsTest$4)\n"
+            + "float_gauge 0.1234000027179718\n"
+            + "# TYPE integer_gauge gauge\n"
+            + "# HELP integer_gauge Generated from Dropwizard metric import (metric=integer.gauge, type=io.prometheus.metrics.instrumentation.dropwizard.DropwizardExportsTest$1)\n"
+            + "integer_gauge 1234.0\n"
+            + "# TYPE long_gauge gauge\n"
+            + "# HELP long_gauge Generated from Dropwizard metric import (metric=long.gauge, type=io.prometheus.metrics.instrumentation.dropwizard.DropwizardExportsTest$3)\n"
+            + "long_gauge 1234.0\n"
+            + "# EOF\n";
+
+    assertThat(convertToOpenMetricsFormat()).isEqualTo(expected);
+  }
+
+  @Test
+  public void testInvalidGaugeType() {
+    Gauge<String> invalidGauge = () -> "foobar";
+
+    metricRegistry.register("invalid_gauge", invalidGauge);
+
+    String expected = "# EOF\n";
+    assertThat(convertToOpenMetricsFormat()).isEqualTo(expected);
+  }
+
+  @Test
+  public void testGaugeReturningNullValue() {
+    Gauge<String> invalidGauge = () -> null;
+    metricRegistry.register("invalid_gauge", invalidGauge);
+    String expected = "# EOF\n";
+    assertThat(convertToOpenMetricsFormat()).isEqualTo(expected);
+  }
+
+  @Test
+  public void testHistogram() {
+    // just test the standard mapper
+    final MetricRegistry metricRegistry = new MetricRegistry();
+    PrometheusRegistry pmRegistry = new PrometheusRegistry();
+    DropwizardExports.builder().dropwizardRegistry(metricRegistry).register(pmRegistry);
+
+    Histogram hist = metricRegistry.histogram("hist");
+    int i = 0;
+    while (i < 100) {
+      hist.update(i);
+      i += 1;
+    }
+
+    // The result should look like this
+    //
+    // # TYPE hist summary
```

Comment:
It works, just that its not a string comparison, hence this comment is to explain why its not asserted like in other cases.

Reply:
can you test it like here: https://github.com/prometheus/client_java/blob/e486e4d0e4a78203a529eea2036320cd7ebaae66/prometheus-metrics-core/src/test/java/io/prometheus/metrics/core/metrics/HistogramTest.java#L950 ?

Reply:
Updated the pr to use 

```
assertThat(textFormat)
        .satisfiesAnyOf(
            text -> assertThat(text).isEqualTo(expected1),
            text -> assertThat(text).isEqualTo(expected2));
```

So that it matches the exact output, and is a much simpler test.